### PR TITLE
Add CRIU enablement flags for jdk8

### PR DESCRIPTION
**Pull request by tajila**
Add CRIU enablement flags for jdk8

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
@keithc-ca Please review these changes

File: closed/OpenJ9.gmk [hunk 1]
```diff
@@ -325,6 +332,12 @@ else
   CMAKE_ARGS += -DJ9VM_OPT_JITSERVER=OFF
 endif # OPENJ9_ENABLE_JITSERVER
 
+ifeq (true,$(OPENJ9_CONFIGURE_CRIU_SUPPORT))
+  CMAKE_ARGS += -DJ9VM_OPT_CRIU_SUPPORT=ON
+else # OPENJ9_CONFIGURE_CRIU_SUPPORT
+  CMAKE_ARGS += -DJ9VM_OPT_CRIU_SUPPORT=OFF
+endif # OPENJ9_CONFIGURE_CRIU_SUPPORT
+
```
Comment (keithc-ca) [inline closed/OpenJ9.gmk:340]: No trailing whitespace, please.

File: jdk/make/closed/autoconf/custom-hook.m4 [hunk 2]
```diff
@@ -119,6 +120,24 @@ AC_DEFUN([OPENJ9_CONFIGURE_COMPILERS],
   AC_SUBST(OPENJ9_DEVELOPER_DIR)
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_CRIU_SUPPORT],
```
Comment (keithc-ca) [inline jdk/make/closed/autoconf/custom-hook.m4:123]: In jdk8, changes to configuration files such as this require re-running `bash common/autoconf/autogen.sh`.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

**Comment by tajila** [comment]
@keithc-ca changes are ready for another look

**Comment by keithc-ca** [comment]
Jenkins compile ppc64le_linux_uma,win32 jdk8

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**